DcCore/DcCore/Extensions/String+Extensions.swift

```diff
@@ -56,5 +56,13 @@ public extension String {
         }
         return attributedText
     }
+}
+
+extension String? {
+    /// returns `true` if `"1"`, in case of `nil` or any other string: return false
```

Comment:
the following makes my mind comments easier to parse:

```suggestion
    /// returns `true` if `"1"`, returns `false` in case of `nil` or any other string
```

i was also thinking about a more descriptive function name (this is generally better than comments), but i did not came to sth better :)

imu, the "numeric" points out somehow that strings as `"true"` are _not_ parse and that it is a bit more special :)